spec/models/enterprise_spec.rb

```diff
@@ -386,33 +386,63 @@
       end
     end
 
-    describe "white label logo link" do
+    describe "white label logo" do
+      let(:enterprise) { build(:enterprise) }
+      let(:content_type) { 'image/png' }
+
       before do
-        # validate white_label_logo_link only if white_label_logo is present
-        allow_any_instance_of(Enterprise).to receive(:white_label_logo).and_return(true)
+        blob = double(
```

Comment:
Could you use an `instance_double` here ? `instance_double` actually check if any stubbed method would existing on a real object, see : https://rspec.info/features/3-13/rspec-mocks/verifying-doubles/instance-doubles/